PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,18 @@
 ## Description
-<!-- Please explain the changes you made here. -->
```

Comment:
The idea behind removing these comments was, as I mentioned in the description, I found it annoying to delete them every time. If they were ghost text, that'd be great, but as far as I can tell, that's not possible in GitHub PR templates. Additionally, I figured if the heading wasn't descriptive enough without a comment, we should probably address that if we can.